code/datums/components/container_craft/craft_operation_datum.dm

```diff
@@ -48,8 +48,10 @@
 	else
 		target_time = recipe.crafting_time
 
-	// Store current inventory state
-	refresh_stored_items()
+	stored_items = list()
+	for(var/requirement_path in recipe.requirements)
+		stored_items[requirement_path] = recipe.requirements[requirement_path] * estimated_multiplier
+
```

Comment:
This change is the reason for #3662 , i tested on local and only modified that part:

<img width="824" height="435" alt="Image" src="https://github.com/user-attachments/assets/b4fc9861-d074-4e73-9280-0d86b764804a" />

It is most likely because of how wildcard_requirements works and it is conflicting

Reply:
For what i could test it also makes so you can't finish meat handpies, it starts and stop cooking, without that code it works fine:
<img width="803" height="491" alt="image" src="https://github.com/user-attachments/assets/5ba3ca7a-bd38-4bdc-a446-c949468f2bc5" />